audformat/core/scheme.py

```diff
@@ -386,6 +390,18 @@ def _check_labels(
                     f"The table '{table_id}' used as scheme labels "
                     "needs to be a misc table."
                 )
+            for column in self._db.misc_tables[table_id].columns.values():
+                if column.scheme_id is not None:
+                    scheme = self._db.schemes[column.scheme_id]
+                    if isinstance(scheme.labels, str):
```

Comment:
Please rebase the master and change to 

```suggestion
                    if scheme.uses_table:
```

Reply:
done